README.md

```diff
@@ -1,12 +1,23 @@
 ## BioCro [![Build Status](https://github.com/ebimodeling/biocro/workflows/R-CMD-check/badge.svg)](https://github.com/ebimodeling/biocro/actions?query=workflow%3AR-CMD-check)
-BioCro is a model that predicts plant growth over time given crop-specific parameters and environmental data as input.
-
-It uses models of key physiological and biophysical processes underlying plant growth ([Humphries and Long, 1995]), and has previously been used for predicting biomass yield and leaf area index of switchgrass and miscanthus ([Miguez et al., 2009]).
-
-BioCro has also been integrated into a suite of tools that link the model directly to crop trait and yield data ([LeBauer et al., 2014]). The Predictive Ecosystem Analyzer ([PEcAn](https://github.com/PecanProject/pecan)) couples BioCro to the [Biofuel Ecophysiological Traits and Yields database](https://www.betydb.org).
+BioCro is a model that predicts plant growth over time given crop-specific
+parameters and environmental data as input.
+
+It uses models of key physiological and biophysical processes underlying plant
+growth ([Humphries and Long, 1995]), and has previously been used for predicting
+biomass yield and leaf area index of switchgrass and miscanthus
+([Miguez et al., 2009]). In 2022, BioCro was reorganized to take a truly modular
+approach to modeling ([Lochocki et al., 2022]) and a new soybean model was
+developed ([Matthews et al., 2022]).
+
+BioCro has also been integrated into a suite of tools that link the model
+directly to crop trait and yield data ([LeBauer et al., 2014]). The Predictive
+Ecosystem Analyzer ([PEcAn](https://github.com/PecanProject/pecan)) couples
+BioCro to the [Biofuel Ecophysiological Traits and Yields
+database](https://www.betydb.org).
```

Comment:
It seems a little odd to me that the reference to PEcAn is to the GitHub repository rather than the project page (https://pecanproject.github.io/), whereas the reference to BETYdb is to the application website (https://www.betydb.org) rather than the GitHub repository (https://github.com/PecanProject/bety).  Also note that https://www.betydb.org seems to not be functioning right now (I get a security warning, followed, if I ignore it, by a 404 error).  The repository sites for PEcAn and BETYdb both refer to their project sites ((https://pecanproject.github.io/ and https://www.betydb.org, respectively), so maybe it's better to give the link to the repository sites in both cases.

Reply:
I just changed the BETYdb link to the github repository.